### ISSUE TITLE
RuntimeError: Unable to make 4067.4 MB space on mps:0.

### ISSUE THREAD

**Issue by cthiele-mogic** [brycedrennan/imaginAIry#347]
I try running

`imagine "a scenic landscape"`

on my Apple M1 pro (Ventura 13.3 with Python 3.10) with 16GB RAM

```
Traceback (most recent call last):
  File "/opt/homebrew/bin/imagine", line 7, in <module>
    imaginairy.cli.imagine.imagine_cmd()
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/opt/homebrew/lib/python3.10/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/opt/homebrew/lib/python3.10/site-packages/imaginairy/cli/imagine.py", line 144, in imagine_cmd
    return _imagine_cmd(
  File "/opt/homebrew/lib/python3.10/site-packages/imaginairy/cli/shared.py", line 176, in _imagine_cmd
    filenames = imagine_image_files(
  File "/opt/homebrew/lib/python3.10/site-packages/imaginairy/api.py", line 61, in imagine_image_files
    for result in imagine(
  File "/opt/homebrew/lib/python3.10/site-packages/imaginairy/api.py", line 176, in imagine
    result = _generate_single_image(
  File "/opt/homebrew/lib/python3.10/site-packages/imaginairy/api.py", line 271, in _generate_single_image
    is_controlnet_model = hasattr(model, "control_key")
  File "/opt/homebrew/lib/python3.10/site-packages/imaginairy/utils/model_cache.py", line 363, in __getattr__
    model = self._mmmw_load_model()
  File "/opt/homebrew/lib/python3.10/site-packages/imaginairy/utils/model_cache.py", line 353, in _mmmw_load_model
    self.__class__._mmmw_cache.set(
  File "/opt/homebrew/lib/python3.10/site-packages/imaginairy/utils/model_cache.py", line 311, in set
    self.make_gpu_space(model_size)
  File "/opt/homebrew/lib/python3.10/site-packages/imaginairy/utils/model_cache.py", line 175, in make_gpu_space
    raise RuntimeError(msg)
RuntimeError: Unable to make 4067.4 MB space on mps:0.
    GPU cache: 0 items; 0.0 MB; Max: 31129.6 MB;
    CPU cache: 0 items; 0.0 MB; Max: 0.0 MB;
    mem_free_total: 3878.8 MB; Ram Free: 6464.7 MB;
```

Should be enough disk space available, any ideas?

**Comment by brycedrennan** [comment]
It's not a disk space issue, its a RAM issue, and it looks like a bug. You can workaround it by either freeing up memory (by closing other software) or switching to an older version of imaginairy.

**Comment by buddhabrot** [comment]
Yeah same issue, there seems to be a bug in memory allocation - I have 32G RAM and it complains.

**Comment by aeroreyna** [comment]
pip install imaginairy==12.0.3 worked for me

**Comment by nervusdm** [comment]
Same problem here.
On windows 1 32gb ram.
And On M1 8GB

**Comment by brycedrennan** [comment]
Should work now in just released 14.0.0
